tests/jest/db/RoutingDAO.spec.js

```diff
@@ -259,6 +261,8 @@ test('RoutingDAO.routeCorridor', async () => {
     { centrelineId: 445100, centrelineType: CentrelineType.SEGMENT },
     { centrelineId: 13455130, centrelineType: CentrelineType.INTERSECTION },
     features[3],
+    { centrelineId: 13455130, centrelineType: CentrelineType.INTERSECTION },
```

Comment:
these changes feel a bit weird - why is it doubling back here? - but we can figure it out as part of our routing updates for #755 .